diff --git a/node_modules/strapi-plugin-content-manager/admin/src/components/CustomTable/Row.js b/node_modules/strapi-plugin-content-manager/admin/src/components/CustomTable/Row.js
index a78e4a1..7a1cd7a 100644
--- a/node_modules/strapi-plugin-content-manager/admin/src/components/CustomTable/Row.js
+++ b/node_modules/strapi-plugin-content-manager/admin/src/components/CustomTable/Row.js
@@ -13,6 +13,7 @@ import { ActionContainer, Truncate, Truncated } from './styledComponents';
 /* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
 
 const dateToUtcTime = date => moment.parseZone(date).utc();
+const dateToLocalTime = date => moment.parseZone(date).local();
 
 const getDisplayedValue = (type, value, name) => {
   switch (toLower(type)) {
@@ -39,7 +40,9 @@ const getDisplayedValue = (type, value, name) => {
       const date =
         value && isObject(value) && value._isAMomentObject === true ? JSON.stringify(value) : value;
 
-      return dateToUtcTime(date).format(DATE_FORMATS[type]);
+      // return dateToUtcTime(date).format(DATE_FORMATS[type]);
+      // return local time zone
+      return dateToLocalTime(date).format(DATE_FORMATS[type]);
     }
     case 'password':
       return '••••••••';
@@ -59,7 +62,6 @@ const getDisplayedValue = (type, value, name) => {
         second,
       };
       const date = moment().set(timeObj);
-
       return date.format(DATE_FORMATS.time);
     }
     default:
